src/OpenTelemetry/Logs/ILogger/OpenTelemetryLoggingExtensions.cs

```diff
@@ -79,6 +82,25 @@ public static class OpenTelemetryLoggingExtensions
                     disposeProvider: false)));
 
         return builder;
+
+        // The warning here is about the fact that the OpenTelemetryLoggerOptions will be bound to configuration using ConfigurationBinder
+        // That uses reflection a lot - so if any of the properties on that class were complex types reflection would be used on them
+        // and nothing could guarantee its correctness.
+        // Since currently this class only contains primitive (boolean) properties this is OK. The top level properties are kept
+        // because the first generic argument is annotated with DAM(All) so will preserve everything on the OpenTelemetryLoggerOptions.
```

Comment:
For my learning - what does DAM stands for here?

Reply:
https://github.com/search?q=repo%3Adotnet%2Fruntime%20DAM&type=code

Reply:
Sorry - I'm so used to shortening `DynamicallyAccessedMembers` as `DAM` (the full attribute name is just too long :-)) - I'll fix this.